bigquery/cloud-client/src/main/java/com/example/bigquery/UpdateDatasetExpiration.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+// [START bigquery_update_dataset_expiration]
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Dataset;
+
+public class UpdateDatasetExpiration {
+
+  public static void runUpdateDatasetExpiration() {
+    // TODO(developer): Replace these variables before running the sample.
+    String datasetName = "my-dataset-name";
+    Long newExpiration = 24 * 60 * 60 * 1000L; // one day in milliseconds
+    updateDatasetExpiration(datasetName, newExpiration);
+  }
+
+  public static void updateDatasetExpiration(String datasetName, Long newExpiration) {
```

Comment:
Please inline newExpiration - doesn't look like it's required for testing (see [Function Structure](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#function-structure))